Resources/Prototypes/_Nuclear14/Entities/Objects/Weapons/Guns/Snipers/snipers.yml

```diff
@@ -125,10 +125,15 @@
     sprite: _Nuclear14/Objects/Weapons/Guns/Snipers/brushgun.rsi
   - type: Item
     sprite: _Nuclear14/Objects/Weapons/Guns/Snipers/brushgun.rsi
+  - type: Gun
+    fireRate: 3
+    soundGunshot:
+      path: /Audio/_Nuclear14/Weapons/Firing/brushgunfire.ogg
```

Comment:
This change doesn't seem related to anything talked about in the pr Readme. Make sure it's intended, and if it is, to talk about what it's actually doing and why.